.github/workflows/ci.yml

```diff
@@ -17,18 +17,27 @@ jobs:
   quick-build:
     if: github.event_name == 'pull_request' && !contains(github.event.pull_request.labels.*.name, 'CI build')
     runs-on: ubuntu-latest
+    container: centos:7
     steps:
       - name: Checkout repository
         uses: actions/checkout@v1
-      - name: Setup Java
-        uses: actions/setup-java@v1
-        with:
-          java-version: 8 
+      - name: Install environment
+        run: |
+          yum -y update
+          yum -y install centos-release-scl-rh epel-release
+          yum -y install java-11-openjdk-devel devtoolset-7 rh-git218 patch perl-Data-Dumper python36-devel python36-numpy python36-pip python36-six
```

Comment:
For the quick build you should only need Java 11 & Maven right? It doesn't need git, devtoolset-7, patch or the python stuff.

Reply:
Oh it was failing when I did not installed `git` but it was probably only because of this [line](https://github.com/karllessard/tensorflow-java/blob/3facbb18831a8ac7ad32dcfe95455a0d2aaadddd/.github/workflows/ci.yml#L38). Let me try that...

Reply:
Ok so only `devtools` was also required, since I'm using `which` in my CI script. I removed other dependencies.